src/components/AddCom.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import { Button } from '@mui/material';
+import { Send } from '@mui/icons-material';
+
+export default function PostForm({ setComments }) {
+    const [ list, setList ] = useState([]);
+    const [ avatar, setAvatar ] = useState('guest');
+    const [ text, setText ] = useState('');
+    const [ msg, setMsg ] = useState(null);
+    const [ error, setError ] = useState(null);
+    const { reviewId } = useParams();
+
+    useEffect(() => {
+        axios.get(`https://gameview.onrender.com/api/users`)
+        .then(({data: {users}}) => {
+            let people = users.map(user => user.username);
+            setList([...people]);
+        })
+    }, [avatar]);
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        setMsg('Comment posted successfully!');
+        setError(null);
+        setText('');
+        axios.post(`https:gameview.onrender.com/api/reviews/${reviewId}/comments`, {
+            username: avatar, body: text,
+        })
+        .then(({data}) => {
+            return data.comment[0];
+        })
+        .then((commentFromApi) => {
```

Comment:
This should be optimistically rendered an dthe setting of state not dependant on the post promise resolving.